gulpfile.js

```diff
@@ -30,8 +30,8 @@ var
     'bower_components/motion-ui/src'
   ];
   cms           = '',
-  themePath     = '',
-  projectName   = '';
+  themePath     = '../themes/custom/',
+  projectName   = 'torsion';
```

Comment:
This variable should be pulled in form the torsion.json file (which gets written by yeoman), we can leave the variables for now, but we'll need to change it so the variables from from that file.